.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # misc
 .DS_Store
+.idea/
```

Comment:
Btw you should add this to your global git ignore, so it effects all your git projects 

https://www.youtube.com/watch?v=oH3MuD6WkZI

Reply:
@eddiejaoude thank you very much! I'll add it. Should I delete this line from .gitignore?

Reply:
It is ok, it can stay for now 👍  Thank you though